### DISCUSSION TITLE
Possibility to report comments, issues, misconduct

### DISCUSSION THREAD

**Discussion by ghost** [community/community#60394] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

On platforms like reddit, facebook, instagram, etc. it's possible report individual comments or interactions of users. Github is kind of social media for developers where harrassment and other types of misconduct can happen. Unfortunately, there is no simple way to report such behavior. One has to click through multiple links in order to get to a form where one can report misconduct but it is by no means straightforward.

What would be great is the ability to block and/or report bad behavior right from the location of such misbehavior (mostly comments, issues, and PRs).

**Comment by gooddavvy** [comment]
Of course there is a way! It's not too hard. Just go to the profile of the person you want to report, then click "Block or Report". A popup will open up to block/report that person. I've used this approach several times.

You can also report from comments by clicking "..." on the comment and then click "Report content".

In summary, you can definitely report or block on GitHub, and it isn't too hard. It is simple to some degree.

Hope that helps!

**Reply by ghost** [reply]
TIL that it's available for some!

Here's an example where it is not available (the comment is fine of course, but there's no "report content" option).

https://github.com/clap-rs/clap/discussions/4749#discussioncomment-5231240

![image](https://github.com/community/community/assets/2829538/7a5bea4a-2424-4f2c-ab70-5e6a60c4f103)

![image](https://github.com/community/community/assets/2829538/cb4dd2da-f2e1-4947-ba99-a843f0b0f79a)

And another example where it's not possible

https://github.com/clap-rs/clap/issues/4989#issuecomment-1609762575
![image](https://github.com/community/community/assets/2829538/271b09a3-a413-49e5-a365-d2b8cf660d4b)


I don't think it's possible to report content in other repositories than github own repos :thinking: 

The only option is going through the user's profile and clicking through a bunch of options (which I don't think is a good user experience)

**Reply by gooddavvy** [reply]
Just report it directly from its profile. For me your discussion text is showing "Report content". My opinion says that it *is* a good user experience:

![image](https://github.com/community/community/assets/110428134/ea256485-74a4-4bbf-a3ec-0c8d951e682a)

**Reply by ghost** [reply]
I'll repeat myself, it's **not** possible on other discussions outside of https://github.com/orgs/community/

**Comment by GhostOf0days** [comment]
It is not bad in its current state. The only issue I have with it is you are redirected to the website rather than a form in the mobile app for the Github mobile app.
